src/components/Rating/Rating.js

```diff
@@ -95,7 +103,7 @@ class Rating extends Component {
   }
 
   render () {
-    const { classes, max, disabled, readOnly, value, onChange } = this.props
+    const { Component, classes, max, disabled, readOnly, value, onChange } = this.props
```

Comment:
Material-UI uses `component` (lower-case "c"). Let's stick to that for consistency.

Reply:
correct!